client/src/components/NavBar.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+const NavBar = () => {
+  return (
+    <nav class="flex items-center justify-between flex-wrap p-6">
+      <div class="flex items-center flex-shrink-0 text-white mr-10">
+        <img
+          align="center"
+          width="160"
+          height="72"
+          src="https://collectiveliberty.org/wp-content/uploads/2020/04/cropped-CollectiveLiberty_FullLogo_01_hi.png"
+        />
+      </div>
+      <ul
+        role="navigation"
```

Comment:
since the navbar is already wrapped with `<nav>`, you don't have to have another role="navigation" down here! everything else looks great